framework/src/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -920,6 +920,15 @@ public Boolean visitWildcard_Array(
     @Override
     public Boolean visitWildcard_Declared(
             AnnotatedWildcardType subtype, AnnotatedDeclaredType supertype, VisitHistory visited) {
+        if (subtype.isUninferredTypeArgument() && supertype.getTypeArguments().isEmpty()) {
```

Comment:
Yes, that makes sense. Just adding the example and some more explanation would be good.